core/src/saros/negotiation/DecompressArchiveTask.java

```diff
@@ -24,29 +25,33 @@
 
   private final File file;
   private final IProgressMonitor monitor;
-  private final Map<String, IProject> idToProjectMapping;
+  private final Map<String, IReferencePoint> idToReferencePointMapping;
   private final String delimiter;
+  private final IReferencePointManager referencePointManager;
 
   /**
    * Creates a decompress task for an archive file that can be executed by {@link IWorkspace#run}.
    * All necessary folders will be created on the fly. <b>Important:</b> Existing files will be
    * <b>overwritten without confirmation</b>!
    *
    * @param file Zip file containing the compressed data
-   * @param idToProjectMapping map containing the id to project mapping (see also {@link
-   *     ISarosSession#getProjectID(saros.filesystem.IProject)}
+   * @param idToReferencePointMapping map containing the id to referencePoint mapping (see also
+   *     {@link ISarosSession#getReferencePointID(IReferencePoint)}
+   *     (saros.filesystem.IReferencePoint)}
```

Comment:
`@param idToReferencePointMapping map containing the id to referencePoint mapping (see also {@link ISarosSession#getReferencePointID(saros.filesystem.IReferencePoint)}`

Reply:
done